src/instruments/src/Common/simVars.tsx

```diff
@@ -59,7 +77,7 @@ const context = React.createContext<{
 });
 const { Provider: InternalProvider }  = context;
 
-type UnitName = string | any; // once typings is next to tsconfig.json, use those units
+export type UnitName = string | any; // once typings is next to tsconfig.json, use those units
```

Comment:
Maybe let's not do this for now, this is more an internal placeholder at this time.

Reply:
changed to `string`